tagger.css

```diff
@@ -14,44 +14,45 @@
     border: 1px solid #909497;
 }
 .tagger input[type="hidden"] {
-  /* fix for bootsrap */
+  /* fix for bootstrap */
   display: none;
 }
 .tagger > ul {
     display: flex;
     width: 100%;
     align-items: center;
-    padding: 4px 0;
+    padding: 4px 5px 0;
     justify-content: space-between;
     box-sizing: border-box;
     height: auto;
-}
-.tagger ul {
+    flex: 0 0 auto;
+    overflow-y: scroll;
```

Comment:
@jcubic, the overflow-y here serves two purposes.

- When wrap: false, the expectation is that the tagger stays on one line, our two options when adding tags longer than the container are to a) expand the container or b) overflow the container. I chose overflow with scroll since there is a good chance that the container cannot / should not be expanded in a layout. 
- When wrap: true, flex wrap will wrap the tags onto new lines as required. The exception here is when a single tag is longer than the width of the container, the scroll is required here to be able to see all of that long tag.

Reply:
Ok, but overflow: scroll doesn't look nice when content is smaller. I would use auto.

Reply:
You're right. I'll update the PR

Reply:
Overflow has been updated to the value of auto. On MacOS these two values are essentially identical as the operating system doesn't show the scroll bars unless they are required.